actions/notification_actions.jsx

```diff
@@ -22,10 +25,30 @@ import {stripMarkdown} from 'utils/markdown';
 
 const NOTIFY_TEXT_MAX_LENGTH = 50;
 
+export function getNotifyLevel(isCrtReply, userSettings, channelSettings) {
+    let notifyLevel = userSettings?.desktop;
+
+    if (isCrtReply) {
+        if ([NotificationLevels.NONE, NotificationLevels.ALL].includes(notifyLevel)) {
```

Comment:
Is there a way to rewrite this logic in a way that makes it easier to follow? Between the multiple returns and the various `||` operators, I'm having trouble figuring out when each setting is used by looking at this code.